compiler/src/iree/compiler/Dialect/Stream/Conversion/HALToStream/Patterns.cpp

```diff
@@ -90,6 +91,11 @@ struct ConvertTensorImportOp
                                                RankedTensorType tensorType,
                                                ValueRange dynamicDims,
                                                OpBuilder &builder) {
+    // If the encoding attr is about packed storage then we don't need all this
```

Comment:
why not? (I know why, but the future reader of this comment does not - explain to them :)

```suggestion
    // If the encoding attr is about packed storage then we don't need all this.
```

Reply:
comments updated.